src/GitLink/Providers/UncProvider.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitTools.Git;
+using System.Text.RegularExpressions;
+
+namespace GitLink.Providers
+{
+    /// <summary>
+    /// A git provider for UNC network paths.
+    /// </summary>
+    public class UncProvider : ProviderBase, IBackSlashSupport
+    {
+        /// <summary>
+        /// A class with placeholder constants.
+        /// </summary>
+        public static class PlaceHolder
```

Comment:
Not sure if embedding this inside a nested class is really necessary. Can you create 2 const members on the UncProvider instead?